### DISCUSSION TITLE
Could I translate it in Chinese and reproduced it on other platforms?

### DISCUSSION THREAD

**Discussion by bringtree** [coqui-ai/TTS#436] [General Q&A] [answered]
https://erogol.com/solving-attention-problems-of-tts-models-with-double-decoder-consistency/
Hello,  Your article is great. Could I translate it into Chinese and reproduced it on other platforms(Wechat)?
I will indicate the original website(https://erogol.com/solving-attention-problems-of-tts-models-with-double-decoder-consistency/)

**Comment by erogol** [comment, marked as answer]
Thx for you interest :)

 feel free to share but please refer to https://coqui.ai/blog/tts/solving-attention-problems-of-tts-models-with-double-decoder-consistency

It is the updated version.
